tools/custom-organization-policy-library/build/custom-constraints/dataproc/dataprocAllowedVpcNetwork.yaml

```diff
@@ -1,11 +1,16 @@
 #@ load("/constraints.lib.star", "build_constraint")
 #@ constraint = build_constraint("dataprocAllowedVpcNetwork")
 
+#@ def condition(vpc_network_uri):
+#@    return "(resource.config.gceClusterConfig.networkUri in " + str(vpc_network_uri) + ") == false"
```

Comment:
@edyw i thinks `str(vpc_network_uri)` is not needed as vpc_network_uri is already a string right ?
We can use directly `vpc_network_uri`

Reply:
Mmm but wait can u remove it completely from the PR so we dont include it (remove the file) so it has part of the generation